exec/java-exec/src/main/java/org/apache/drill/exec/work/WorkManager.java

```diff
@@ -59,12 +61,14 @@
 public class WorkManager implements AutoCloseable {
   private static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(WorkManager.class);
 
+  public static final long EXIT_TIMEOUT = 30000L;
```

Comment:
`30000L` --> `30_000L` for quicker reading.

Reply:
Done